modules/cnv-edit-cdrom-vm.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-edit-cdrom_{context}"]
+
+= Editing CD-ROMs for {object-gui}
+Use the following procedure to configure CD-ROMs for virtual machines.
+
+.Procedure
+
+. From the *{object-gui}* tab, select your {object}.
+. Select the *Overview* tab.
+. Click the pencil icon to the right of the *CD-ROMs* label to open the *Edit CD-ROM* window.
+* If no CD-ROMs are available for editing, the CD label initially displays *blank*.
+* If there are CD-ROMs avaiable, you can eject the CD-ROM by clicking *Eject CD-ROM* or remove it by clicking *-*.
+. In the *Edit CD-ROM* window, do the following:
+.. Select the type of CD-ROM configuration in the *Media Type* field.
+* On Windows systems, *Windows guest tools* is attached by default in
+the *Media Type* field.
+.. Complete the required information for each *Media Type*.
+.. Click *Add CD-ROM*.
+. When all CD-ROMs are added, click *Save*.
+
+// Scrubbing all conditionals used in module
+
+:object!:
```

Comment:
@bgaydosrh 
@ousleyp 
@aburdenthehand 

Shouldn't these be unset the same way they were set? Inside the conditionals? These is a very small chance that they could unset similarly named variables set by someone else.